frontend/javascripts/oxalis/model/sagas/update_actions.ts

```diff
@@ -220,18 +256,23 @@ export type ServerUpdateAction =
   | AsServerAction<DeleteEdgeUpdateAction>
   | AsServerAction<UpdateSkeletonTracingUpdateAction>
   | AsServerAction<UpdateVolumeTracingUpdateAction>
-  | AsServerAction<UpdateUserBoundingBoxesAction>
-  | AsServerAction<CreateSegmentVolumeAction>
-  | AsServerAction<UpdateSegmentVolumeAction>
-  | AsServerAction<DeleteSegmentVolumeAction>
+  | AsServerAction<UpdateUserBoundingBoxesUpdateAction>
+  | AsServerAction<CreateSegmentUpdateAction>
+  | AsServerAction<UpdateSegmentUpdateAction>
+  | AsServerAction<DeleteSegmentUpdateAction>
   | AsServerAction<UpdateBucketUpdateAction>
   | AsServerAction<UpdateTreeVisibilityUpdateAction>
   | AsServerAction<UpdateTreeGroupVisibilityUpdateAction>
   | AsServerAction<RevertToVersionUpdateAction>
   | AsServerAction<UpdateTreeGroupsUpdateAction>
-  | AsServerAction<CreateTracingUpdateAction>
-  | AsServerAction<RemoveFallbackLayerAction>
-  | AsServerAction<UpdateTdCameraAction>;
+  | AsServerAction<RemoveFallbackLayerUpdateAction>
+  | AsServerAction<UpdateTdCameraUpdateAction>
+  | AsServerAction<UpdateMappingNameUpdateAction>
+  | AsServerAction<SplitAgglomerateUpdateAction>
+  | AsServerAction<MergeAgglomerateUpdateAction>
+  // These two actions are never sent by the frontend and, therefore, don't exist in the UpdateAction type
```

Comment:
what would happen if they are added to `UpdateAction`? Would this force us to handle the actions during diffing etc? maybe that would be a bit more consistent, since the current way seems a bit confusing (e.g., only looking at the `UpdateAction` definition might raise the question why not all actions are listed there). I don't have a strong opinion about this, though.

Reply:
A quick test showed that nothing would happen, the code still type-checks and we wouldn't need to handle the actions during diffing, since the diffing is where update actions are instantiated in the first place.
I see this more as a distinction to _help_ readers of the code. I wanted to avoid that they wonder why there was no function to create such an update action (at least for the `ImportVolumeTracingUpdateAction` - the `CreateTracingUpdateAction` is created on demand for the version view).
I would argue that the current definitions more closely resemble the reality, but if it's confusing I'm also open to adapt it the way you described.

Reply:
Ok, I understand. Then, let's keep it as is :)